lib/tmp.js

```diff
@@ -349,10 +349,10 @@ function _prepareTmpFileRemoveCallback(name, fd, opts) {
       fs.closeSync(fdPath[0]);
     }
     catch (e) {
-      // under some node/windows related circumstances, a temporary file 
+      // under some node/windows related circumstances, a temporary file
       // may have not be created as expected or the file was already closed
       // by the user, in which case we will simply ignore the error
-      if (e.errno != -_c.EBADF && e.errno != -c.ENOENT) {
```

Comment:
@raszi AFAIK nodejs always used negative values but it can be that behavior has been changed... - therefore my initial Math.abs(), you remember?